hospital/src/main/java/by/emel/anton/service/UserServiceResolverImp.java

```diff
@@ -0,0 +1,28 @@
+package by.emel.anton.service;
+
+import by.emel.anton.terminalprog.AnswerType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class UserServiceResolverImp implements UserServiceResolver{
+
+    private final Map<AnswerType, UserService> map;
+
+    @Autowired
+    public UserServiceResolverImp(
+            @Qualifier("FromHibernate") UserService hibernateUserService) {
```

Comment:
Qualifier should be named like specific type of service. Say `HibernateUserService`. Apply this to all qualifiers

Reply:
done